src/WebUI/Views/Shared/_Layout.cshtml

```diff
@@ -34,6 +42,14 @@
         <a class="header-slide--item" href="#">Placeholder 2</a>
         <a class="header-slide--item" href="#">Placeholder 3</a>
         <div class="header-slide--separator"></div>
+        @if (User.Identity.IsAuthenticated)
```

Comment:
Should this be the other way round? If the user is authenticated, we should show a sign-out link; if not, it should be a sign-in.

Reply:
I deep tested this. I switched the two lines. Checked that after I login there is "IsAuthenticated" true.  Saw that the buttons show up correctly. 

A bit weird but both cases just worked (IsAuthenticaed showed sign in, !IsAuthenticated showed up sign in)

Anyway I don't see a problem after https://github.com/codidact/core/pull/29/commits/47e2c020e2c85c1b7a0ba7d6e9c5f50277bb6c2b